crates/engine/schema/src/builder/extension/link_directive.rs

```diff
@@ -0,0 +1,145 @@
+use std::collections::HashMap;
```

Comment:
This module is just copy-pasted and pared down to just what the schema builder needs.